### PR TITLE
Try to fix travis (take 1)

### PR DESCRIPTION
**Pull request by acinader**
1. use my github token (i think)
2. only release on tags

see: #1482

### CODE REVIEW

**Comment by acinader** [comment]
@noobs2ninjas if you want to tag team on this we can use slack?  I have NO IDEA how the build works for this repo, but i do have some experience muddling through travis configs between the two of use, maybe....

File: .travis.yml [hunk 1]
```diff
@@ -22,7 +23,7 @@ jobs:
       deploy:
         - provider: releases
           api_key:
-            secure: $GITHUB_TOKEN # Set in travis-ci.org dashboard
+            secure: m7YayvmctH5lyHj13EV2j898P5xfEfN/ZvFm2gL/xveTNOGaET1O4ZmGmHApxC2QwIZvtwmO3U9pmB4XVVQ7kIlTsl01h84y8gx1XQqEKoQINcaVaNCsh9EdxAqj08aiwe/1dp8W8rI/ptOrxzWJDWcxKx5tMIbBzctGeL3GDKphVvD44s43oosqQCKzP+qHiI6rRRxu+0QccdbTagwd0X/yaLZ0IbTo35hWFEyJ6NfJnmCW/lNwWj86YksLCZx0eKdxUSQlIhnm0upTyowudt9bv8vSdEYi2AtRKoana/X53593O8RAVEc+S6NgMmeQ1/ZD1eDeyYLzN/z21EHb4yiwTnMUUhDBMTn8KKTiiSVDQOfAyRwtKF+1qN2iDr9Dh3A8+dCxOxJJpIPImZimt8KMCKB9WTtnJhvIoMCDpc8l29XM/oBxjCiBFxPClb9CZ74T5b+xl+yIqy2u8RiaZ1Va5vmWjMSXDsGHoZ0rJTOBkBBhyr+47yIw2VeY2z2q5hiALV3DnE5Esxyddhw7OXNZcvKriujnxcTOJwphm4XBMI/3rSH27uz7AaNdUHfTwaKqimPR06Kdv65hKmFuA5p/GuRdSYfYASvXYYrPjqKkst1FP6sWjJtpXvZIT/06RXDbGXZ6LB3TFHpOahPy8ID746OyBKgWzOB3ZCNmFCk=
```
Comment (drdaz) [inline .travis.yml]: This, I mean. Our keys shouldn't be here.

**Comment by drdaz** [comment]
@acinader This seems like a security issue; anybody can see this page. Which means anybody has access to this GitHub token, and whatever rights it gives on our repo. Am I missing something?

**Comment by acinader** [comment]
Hi @drdaz  -- that is not the key.  It is an encrypted hash of my key.

see: https://docs.travis-ci.com/user/encrypting-files

**Comment by drdaz** [comment]
Ahh that's awesome! TIL.

panic_mode = false

**Pull request merged**